tmh_registry/registry/api/serializers.py

```diff
@@ -0,0 +1,91 @@
+from rest_framework import serializers
+
+from ..models import Hospital, Patient, PatientHospitalMapping
+
+
+class HospitalSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Hospital
+        fields = ["id", "name", "address"]
+
+
+class PatientSerializer(serializers.ModelSerializer):
```

Comment:
If using a single serializer cause issues, in Swagger for example, maybe we should consider using 2 serializers, for read and write operations

Reply:
Swagger is an issue indeed with this implementation, I will try to change it to 2 serializers as you suggest.